### DISCUSSION TITLE
Missing available Python version support on the PyPi page

### DISCUSSION THREAD

**Discussion by RoieTBD** [aio-libs/aiohttp#7771] [Q&A] [answered]
On [aiohttp's PyPi page](https://pypi.org/project/aiohttp/), there's a Programming Language list and it is missing Python 3.11 which someone told me aiohttp supports just fine as a release version (I haven't checked for myself). Are there any plans for adding it to the list? I'd like to see it getting updated for Python 3.12, too, when a release version support comes. PyPi supports listing up to even Python 3.13, so there should be no problem.

**Comment by webknjaz** [comment, marked as answer]
It does not list the Trove classifiers because the corresponding CI jobs aren't green. So we don't put the label on just yet, until somebody fixes that.
